src/PokemonROMEditor/PokemonROMEditor.csproj

```diff
@@ -5,7 +5,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{B2C84D30-3766-452A-9EA2-6BAFBCDEE41C}</ProjectGuid>
-    <OutputType>WinExe</OutputType>
+    <OutputType>Exe</OutputType>

```

Comment:
This is the only part that I'm not sure of and that is only because I'm not experienced enough with csproj configurations. I will clone this down and check it out and if it still works okay I will go ahead and approve this PR.

Reply:
Maybe a few weeks ago I would be better suited to answer what this change was for, but it is quite possible that I accidentally changed a setting in Visual Studio that resulted in this. Feel free to ignore it if it doesn't work in other machines.

Reply:
Okay I pulled down the changes and tested it out. Looks like this change makes it launch a console which then launches the actual program. If you are willing to submit another commit that removes the changes to this file then the pull request will be good to go.